### PR TITLE
[tldr] add tldr command

### PR DESCRIPTION
**Pull request by obviyus**

### CODE REVIEW

File: api/tldr.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    if update.message:
+        message: 'telegram.Message' = update.message
+    else:
+        return
+
+    text: str
+    content: str
+
+    try:
+        content: str = message.reply_to_message.text or message.reply_to_message.caption  # type: ignore
```
Comment (Tyagdit) [inline api/tldr.py]:
```suggestion
        content = message.reply_to_message.text or message.reply_to_message.caption  # type: ignore
```

File: api/tldr.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    try:
+        params.update({"SM_URL": match.group("url")})
+        r = requests.post(url=base_url, params=params).json()
+    except AttributeError:
+        content = content.replace('\r', '').replace('\n', '')
+
+        sentences: int = content.count(".")
+        if sentences <= 3:
+            message.reply_text(text="Content too short.")
+            return
```
Comment (Tyagdit) [inline api/tldr.py]: Does the API not return an error for this?

File: api/tldr.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+        data: Dict[str, str] = {"sm_api_input": content}
+        logging.error(content)
```
Comment (Tyagdit) [inline api/tldr.py]:
```suggestion
```

**Review by Tyagdit — CHANGES_REQUESTED** [review]

File: api/tldr.py (hunk 2 above)
Comment (obviyus) [reply api/tldr.py]: It does with a simple `CONTENT TOO SHORT` message. I'd rather avoid making that call entirely (considering we're limited to 100 calls/day).

**Review by Tyagdit — APPROVED** [review]

**Pull request merged**